Lib/http/server.py

```diff
@@ -443,7 +445,7 @@ def handle(self):
         while not self.close_connection:
             self.handle_one_request()
 
-    def send_error(self, code, message=None, explain=None):
+    def send_error(self, code, message=None, explain=None, *, range_size=None):
```

Comment:
Since subclassing is a big use case and this is a publicly documented method, adding a parameter like this is a problem for backwards compatibility with existing overridden implementations. Any new parameter should be added to the main module RST documentation, and I suppose also listed under Porting at <https://docs.python.org/dev/whatsnew/3.14.html#porting-to-python-3-14>.

If we do add a parameter, I’d prefer something more general like a _header_fields_ list, and have the caller pass [('Content-Range', f'bytes */{range_size}')].